### DISCUSSION TITLE
Can't view demo event

### DISCUSSION THREAD

**Discussion by jaymeklein** [pretalx/pretalx#1920] [Q&A] [answered]
Hello everyone, I'm struggling a bit trying to setup the development environment, and I don't know what is wrong.

First of all, i followed these [instructions](https://docs.pretalx.org/developer/setup/), up to the `python manage.py runserver` command, which run properly and got the front and backend up and running.

Even created the test event:
`python manage.py create_test_event`

![image](https://github.com/user-attachments/assets/649b886b-9711-490a-8fbb-e7a99ff6510a)

The problem comes when i click the "View Event" button:
![image](https://github.com/user-attachments/assets/ea8311d6-9e52-4823-a224-4d28783ca9b9)

It gives me a 404 error:
![image](https://github.com/user-attachments/assets/8d326152-6fca-4bb0-9ad3-1edd09e15e6c)

Shouldn't it redirect me to the event page?

Things i've tried:
- Clone the project again;
- Create a new virtual environment;
- Changing OS (At first i was running in WSL, then switched to W11, both with the same problem);

Am i doing something wrong here?
I'm willing to contribute to this project,

**Comment by rixx** [comment]
I don't think the problem is with your setup – this is likely a configuration problem instead.

Things to try:

- can you access http://127.0.0.1:8000/democon/ (rather than the /schedule path)?
- is there a custom domain configured in the event settings?

**Comment by jaymeklein** [comment, marked as answer]
Hey there!

Nope, accessing `/democon` directly also gives that error:
![image](https://github.com/user-attachments/assets/c69af36a-601a-4404-9d9c-3f9edfa1dd78)

BUT...
The custom domain gave me a hint:
![image](https://github.com/user-attachments/assets/b4bb3bb7-b8c9-4d27-a009-d1184acaa2c6)

The `View Event` button redirected me to http://127.0.0.1:8000/democon rather than http://localhost:8000/democon

Acessing the localhost works fine:
![image](https://github.com/user-attachments/assets/b10fcd48-68e7-4ae7-8431-81014be844dc)

Thanks!
